candlepin.spec.tmpl

```diff
@@ -44,8 +44,9 @@ BuildRequires: selinux-policy-doc
 
 Requires: java-17 >= 1:17.0.0
 Requires: wget
-%if 0%{?fedora}
-Requires: tomcat
+# TODO: remove the version restriction on Tomcat once CP supports the Java EE namespace changes
+%if 0%{?fedora} or 0%{?rhel} >= 9
```

Comment:
`or` is not an rpm spec operator. this should be `||`

Reply:
done